app/mailers/passwordless/mailer.rb

```diff
@@ -18,7 +18,7 @@ def magic_link(session)
       email_field = @session.authenticatable.class.passwordless_email_field
       mail(
         to: @session.authenticatable.send(email_field),
-        subject: 'Your magic link ✨'
+        subject: I18n.t('passwordless.mailers.subject')
```

Comment:
I think `passwordless.mailer.subject` (singular `mailer`) is closer to the actual structure. Also – we only have one :)